src/utils/omit.ts

```diff
@@ -0,0 +1,8 @@
+import * as Lodash from 'lodash';
+
+export type Diff<T extends string, U extends string> = ({[P in T]: P } & {[P in U]: never } & { [x: string]: never })[T];
```

Comment:
TDiff?

Reply:
Don't you think it's nicer like this, because it blends in amongst `Pick<>` and co?

Reply:
Makes sense indeed, could expose those as types BTW?

Reply:
You mean move them to the `types` directory?

Reply:
Yes

Reply:
Done